public/electron.js

```diff
@@ -129,6 +129,24 @@ function createWindow() {
   });
 }
 
+ipcMain.on('restart-lnd-process', async event => {
```

Comment:
Changin this to 'lnd-restart-process' would allow us to keep the regex filter the same. We should migrate other event names too, but no rush on that.

Reply:
sounds good, fixed